.pre-commit-config.yaml

```diff
@@ -38,3 +38,9 @@ repos:
           - types-PyYAML==6.0.12
           - types-requests==2.28.11
           - types-setuptools==64.0.1
+  - repo: https://github.com/pre-commit/pygrep-hooks
+    rev: v1.9.0
+    hooks:
+      - id: rst-backticks
+      - id: rst-directive-colons
+      - id: rst-inline-touching-normal
```

Comment:
Nice, I guess these are quite fast and don't slow down the pre-commit hooks much right? It might make sense to fix the other issues in a separate commit if you want to already add this in the PR here and get green CI.

Reply:
I don't think those additional checks slow down the pre-commit. LMK if you notice a degradation in CI performance.
Fixed the issues discovered by the new checks.

Reply:
Just gave it a quick try locally and the new hooks are much faster than all our existing linters even when running all files, so looks good :)